.travis.yml

```diff
@@ -67,6 +69,12 @@ script:
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then bundle exec bundle-audit check --update --ignore CVE-2015-9284; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then yarn build-storybook; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "2" ]; then bin/test-console-check; fi
+  - yarn e2e:ci
+  # Dropping and recreating the database for Capybara/rspecs
+  - export FOREM_OWNER_SECRET="secret"
+  - bundle exec rails db:drop
```

Comment:
I'm not sure if this is necessary, but I'm dropping and recreating the db and loading the schema for the rspecs/capybara tests.